addons/trenches/functions/fnc_isTrench.sqf

```diff
@@ -0,0 +1,22 @@
+/*
+ * Author: chris579
+ * Determines whether an object is a trench.
+ *
+ * Arguments:
+ * 0: Object <OBJECT>
+ *
+ * Return Value:
+ * Is trench <BOOL>
+ *
+ * Example:
+ * [myObj] call ace_trenches_fnc_isTrench
+ *
+ * Public: No
```

Comment:
Since it's not a public function anyway, you could just make this a macro

Reply:
Or this could be public. Dunno if there is any useful case for such a public function.